pyproject.toml

```diff
@@ -0,0 +1,3 @@
+[tool.black]
+line-length = 88
+target-version = ['py39']
```

Comment:
Actually, should we implement an 'newline at end of the file' linting rule or something? I see quite some of these changes based on the IDE settings people use

Reply:
I don't understand how that's possible. We have this rule and it trips the build too.
<img width="408" alt="Screenshot 2023-08-27 at 14 51 36" src="https://github.com/Vandebron/mpyl/assets/1911436/bdc7e5b6-fada-465e-b8c7-a6ab5e5dc4f3">